README.rst

```diff
@@ -40,16 +59,19 @@ Features
 * Digital signatures
 * Secret-key encryption
 * Public-key encryption
-* HMAC (coming soon)
 
 
 Changes
 -------
 
-* master: PyNaCl has been ported to use the new APIs available in cffi 1.0+.
-  Due to this change we no longer support PyPy releases older than 2.6.
+* 0.4.0:
+
+  * PyNaCl has been ported to use the new APIs available in cffi 1.0+.
+    Due to this change we no longer support PyPy releases older than 2.6.
+
+  * Python 3.2 support has been dropped.
 
-* master: Python 3.2 support has been dropped.
+  * ed25519 to curve25519 functions have been added.
```

Comment:
I'd rephrase this as "Functions to convert between Ed25519 and Curve25519 keys have been added"